src/defaults_vs_conda-forge.rst

```diff
@@ -0,0 +1,47 @@
+conda defaults and conda-forge are incompatibilities
+====================================================
+
+It is quite common to install a package from conda-forge and,
+when trying to use it,
+see an error like (OS X example):
+
+.. code-block:: shell
+
+    ImportError: dlopen(/Users/userX/miniconda3/envs/problem/lib/python3.4/site-packages/rpy2/rinterface/_rinterface.so, 2): Library not loaded: @rpath/libicuuc.54.dylib
+      Referenced from: /Users/userX/miniconda3/envs/problem/lib/python3.4/site-packages/rpy2/rinterface/_rinterface.so
+      Reason: image not found
+
```

Comment:
Worth to add Linux example as well so people doing a web search for the error message might find their way here too? 

```
.. code-block:: shell

    ImportError: libicuuc.so.54: cannot open shared object file: No such file or directory
```